apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -242,16 +242,23 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
               errorMessage={formik.errors.eventTitle}
               svgIcon={<FontAwesomeIcon className="input-icon" icon={faPencilAlt} />}
             />
-            <CustomTextInput
-              title="Description"
-              fieldName="description"
-              value={formik.values.description}
-              onChange={formik.handleChange}
-              hasErrors={formik.touched.description && !!formik.errors.description}
-              errorMessage={formik.errors.description}
-              svgIcon={<FontAwesomeIcon className="input-icon" icon={faAlignLeft} />}
-            />
-
+            <div className="textarea-create-event-container">
+              <label className="description-label" htmlFor={'description'}>
+                Description
+              </label>
+              <div className="textarea-sub-container">
+                <textarea
+                  id="description"
+                  className={`description-msg ${
+                    formik.touched.description && !!formik.errors.description ? 'error' : ''
+                  }`}
+                  rows={1}
+                  value={formik.values.description}
+                  onChange={formik.handleChange}
+                />
+              </div>
+              <FontAwesomeIcon className="description-input-icon" icon={faAlignLeft} />
+            </div>
```

Comment:
💡Just a note but this could be turned into a component if it ends up being used in multiple places, like for the edit profile page for example